torch/testing/_internal/distributed/_tensor/common_dtensor.py

```diff
@@ -76,6 +76,7 @@ class ModelArgs:
     dropout_p: float = 0.1
     use_attn_mask: bool = True
     weight_tying: bool = True
+    checkpoint_activations: bool = False
```

Comment:
Is it okay to add this arg to the transformer's `ModelArgs` for an option for inline activation checkpointing?

cc: @tianyu-l @wanchaol